portal-frontend/src/app/features/applications/application-details/cove-details/cove-details.component.html

```diff
@@ -0,0 +1,76 @@
+<div *ngIf="_applicationSubmission" class="review-table">
+  <div class="subheading2 full-width">Provide the name and contact information for all transferees</div>
+  <div class="full-width transferee-table">
+    <div class="subheading2">Type</div>
+    <div class="subheading2">Full Name</div>
+    <div class="subheading2">Organization Name</div>
+    <div class="subheading2">Phone</div>
+    <div class="subheading2">Email</div>
+    <ng-container *ngFor="let transferee of transferees">
+      <div>{{ transferee.type.code === 'INDV' ? 'Individual' : 'Organization' }}</div>
+      <div>{{ transferee.displayName }}</div>
+      <div>
+        {{ transferee.organizationName }}
+        <app-no-data [showRequired]="false" *ngIf="!transferee.organizationName"></app-no-data>
+      </div>
+      <div>
+        <span *ngIf="transferee.phoneNumber">{{ transferee.phoneNumber | mask : '(000) 000-0000' }}</span>
+      </div>
+      <div>{{ transferee.email }}</div>
+    </ng-container>
+  </div>
+  <app-no-data class="full-width" [showRequired]="showErrors" *ngIf="transferees.length === 0"></app-no-data>
+
+  <div class="subheading2 grid-1">How many hectares will the covenant impact?</div>
+  <div class="grid-double">
+    {{ _applicationSubmission.coveAreaImpacted }}<span *ngIf="_applicationSubmission.coveAreaImpacted"> ha</span>
+    <app-no-data [showRequired]="showErrors" *ngIf="!_applicationSubmission.coveAreaImpacted"></app-no-data>
+  </div>
+
+  <div class="subheading2 grid-1">What is the purpose of this covenant?</div>
+  <div class="grid-double">
+    {{ _applicationSubmission.purpose }}
+    <app-no-data [showRequired]="showErrors" *ngIf="!_applicationSubmission.purpose"></app-no-data>
+  </div>
+
+  <div class="subheading2 grid-1">Explain how the covenant impacts the use of agricultural land for farm purposes.</div>
+  <div class="grid-double">
+    <span *ngIf="_applicationSubmission.coveFarmImpact !== null">
+      {{ _applicationSubmission.coveFarmImpact }}
+    </span>
+    <app-no-data [showRequired]="showErrors" *ngIf="!_applicationSubmission.coveFarmImpact"></app-no-data>
```

Comment:
nitpick: applicationSubmission in covenant, I know that covenant is a type of applicationSubmission but "covenant"  instead looks less wordy

Reply:
Gonna leave it alone since I think its better to have the proposals all be consistent.